assets/js/translator.js

```diff
@@ -4,6 +4,7 @@ var curSrcLang, curDstLang;
 var recentSrcLangs = [], recentDstLangs = [];
 var droppedFile;
 var textTranslateRequest;
+var recaptchaScriptSrc = 'https://www.google.com/recaptcha/api.js?onload=recaptchaRenderCallback&render=explicit&hl=';
```

Comment:
Use exported. See comments in other files.

On Aug 27, 2016 7:21 PM, "Kira Droganova" <notifications@github.com> wrote:

> In assets/js/translator.js
> <https://github.com/goavki/apertium-html-tools/pull/72#discussion_r76523674>
> :
>
> > @@ -4,6 +4,7 @@ var curSrcLang, curDstLang;
> >  var recentSrcLangs = [], recentDstLangs = [];
> >  var droppedFile;
> >  var textTranslateRequest;
> > +var recaptchaScriptSrc = 'https://www.google.com/recaptcha/api.js?onload=recaptchaRenderCallback&render=explicit&hl=';
>
> localization.js uses it
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/goavki/apertium-html-tools/pull/72/files/0695d3a91697696ecbed646aaabd0ce0948ce563#r76523674>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEBEfk5MdCEzzWhvPCbE5pl2Po_HjJwXks5qkMZsgaJpZM4Jux7N>
> .
>